hardhat.config.js

```diff
@@ -59,7 +59,7 @@ const withOptimizations =
 
 const ACCOUNTS_NUMBER = parseInt(process.env.USERS_NUMBER);
 
-const web3 = new Web3(new Web3.providers.HttpProvider(process.env.PROXY_URL));
+const web3 = new Web3(new Web3.providers.HttpProvider(process.env.PROXY_URL, 3000000));
```

Comment:
why do we need it? have we ever fallen here?
if you want to add one more timeout expiration value, wouldn't it be better to create an entry in .env files or move this setting into a constant class?

Reply:
We can remove this. I set this when found some timeout which we can increase